xyz-hub-service/src/main/java/com/here/xyz/httpconnector/config/JDBCExporter.java

```diff
@@ -165,7 +165,7 @@ private static Future<Export.ExportStatistic> executeParallelExportAndCollectSta
                                         .withRowsUploaded(overallRowsUploaded)
                                 );
                             }else{
-                                logger.warn("[{}] Export failed {}: {}", j.getId(), j.getTargetSpaceId(), t.cause());
+                                logger.warn("job[{}] Export failed {}: {}", j.getId(), j.getTargetSpaceId(), t.cause());
```

Comment:
Please remove the third occurrence of `{}` otherwise the stack-trace of t will be missing in the logs.